src/main/scala/org/constellation/consensus/ConsensusManager.scala

```diff
@@ -135,15 +142,16 @@ class ConsensusManager[F[_]: Concurrent](
   def createRoundData(roundId: RoundId): F[(RoundData, Seq[(Transaction, Int)], Seq[(ChannelMessage, Int)])] =
     for {
       transactions <- transactionService.pullForConsensus(dao.minCheckpointFormationThreshold)
-
       facilitators <- LiftIO[F].liftIO(dao.readyFacilitatorsAsync)
       tips <- concurrentTipService.pull(facilitators)(dao.metrics)
-      _ <- if (tips.isEmpty) Sync[F].raiseError[Unit](NoTipsForConsensus(roundId, transactions.map(_.transaction.hash)))
+      _ <- if (tips.isEmpty)
```

Comment:
it can be a validation function "validateEmptyTips" or something like this, i feel the block "if else" is difficult to read if it is contained in the "for yield" block, because we have next level in one function

if I am looking for a given piece of code, I don't want to read the inside of the function, I want to read the name that tells me what to do and move on

Reply:
yep, can be, but let's keep such improvements in different PRs than functionalities